src/additional_services/assertoor.star

```diff
@@ -0,0 +1,54 @@
+HTTP_PORT_NUMBER = 8080
+
+ASSERTOOR_IMAGE = "ethpandaops/assertoor:latest"
```

Comment:
Generally it's safer to specify a commit or version tag. We don't want it to break if `latest` is updated to something with breaking changes.